.github/workflows/cd.packages-stable.create-release-drafts.yml

```diff
@@ -63,7 +63,16 @@ jobs:
           registry-url: https://registry.npmjs.org/
 
       - name: Check package versions
-        run: ./tasks/check-package-version.sh subgraph SHOULD_PUBLISH_SUBGRAPH SUBGRAPH_NEW_VERSION >> "$GITHUB_ENV"
+        run: |
+          SUBGRAPH_NEW_VERSION=$(jq -r .version packages/subgraph/package.json)
+          if git tag | grep -e "^subgraph" | grep v"$SUBGRAPH_NEW_VERSION";then
```

Comment:
could also just be more strict: `git tag | grep -e "^subgraph@v${SUBGRAPH_NEW_VERSION}$"`

Reply:
done